_scripts/build_person_index.py

```diff
@@ -86,7 +89,10 @@ def find_wiki_title(person):
 
 
 for person in ynmp_export['persons']:
-    
+    if not person['standing_in'] or '2015' not in person['standing_in']:
```

Comment:
Ahh, good point! But actually, I didn’t mean to commit this bit at all! The plan is to output data for everyone, and then handle it in the templates.